### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by violog**

### CODE REVIEW

**Comment by violog** [comment]
Adding more features is in progress

**Comment by Arvolear** [comment]
Let's add proper linting for solidity

File: test/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  });
+}
+
+export function cleanAfterEach() {
+  // Manual removing of generated files should be faster than full cleanup because of the frequent re-compilations
+  // Another way is to use the different outdir for each case
+  // Surprisingly, full cleanup after each case is faster than manual rm of outir
+  afterEach(async function () {
+    await this.env.run(TASK_CLEAN);
+    // rmSync(this.outdir, { recursive: true, force: true });
```
Comment (Arvolear) [inline test/helpers.ts:37]: Please remove the comment

**Comment by Arvolear** [comment]
Let's add a CI

**Review by Arvolear — APPROVED** [review]
Ok

**Pull request merged**